extensions/3DTILES_bounding_volume_S2/0.0.0/README.md

```diff
@@ -1,75 +1,92 @@
+
+## Bounding Volume
+
+This extension enables using S2 cells and a set of bounding heights to describe a bounding volume in 3D Tiles. The `token` represents an S2 cell. The cell describes 4 vertices on the surface of the WGS84 ellipsoid for the area, and the `minimumHeight` and `maximumHeight` describe the vertical limits with respect to the WGS84 ellipsoid.
 
-## Cell Token
+![Volume](figure/../figures/volume.jpg)
 
-This extension uses tokens, or hexadecimal string representations of `S2CellId` for two reasons:
+<center>S2 cell (<tt>"89c6c7"</tt>) covering the Philadelphia Center City area, with minimum height set to <tt>0</tt> m and maximum height set to <tt>1000</tt> m.</center>
+
+### Cell Token
+
+This extension uses tokens, or hexadecimal string representations of S2 cell identifier for two reasons:
  1. Precision: Using a token will require a client to convert it to the correct data type: `uint64`
- 2. Conciseness: Tokens provide a more concise representation of `S2CellId`s
+ 2. Conciseness: Tokens provide a more concise representation of identifiers
 
 More details on computing an `S2CellToken` can be found in the [S2 reference implementation](https://github.com/google/s2-geometry-library-java/blob/c28f287b996c0cedc5516a0426fbd49f6c9611ec/src/com/google/common/geometry/S2CellId.java#L468).
 
-## Heights
+### Bounding Heights
 
-The S2 cell itself is used to specify an area on the surface of the ellipsoid. To create a bounding volume, the `minumumHeight` and `maximumHeight` properties must be specified. These heights must be specified in meters above the WGS84 ellipsoid.
+The S2 cell itself is used to specify an area on the surface of the ellipsoid. To create a bounding volume, the `minimumHeight` and `maximumHeight` properties must be specified. These heights must be specified in meters above the WGS84 ellipsoid.
 
 ## Subdivision
 
 The S2 library defines a [cell hierarchy](http://s2geometry.io/devguide/s2cell_hierarchy), that follows uniform subdivision using a quadtree structure, where each cell subdivides into 4 smaller cells that combine to occupy the same area as the parent.
 
 When used with `3DTILES_implicit_tiling`, a `QUADTREE` subdivision scheme will follow the rules for subdivision as defined by S2. When an `OCTREE` subdivision scheme is used, the split in the vertical dimension occurs at the midpoint of the `minimumHeight` and `maximumHeight` of the parent tile. The `availability` bitstreams are ordered by the Morton index of the tile, as specified by `3DTILES_implicit_tiling`, not by the Hilbert index used by S2. Additionally, the `maximumLevel` property cannot be greater than `30 - {Level of root S2CellId}` because S2 cell hierarchy only extends to level 30.
 
-| Cell  | Quadtree Subdivsion | Octree Subdivsion |
+| Cell  | Quadtree Subdivision | Octree Subdivision |
 |---|---|---|
 | ![Parent Cell](figures/parent.png)  | ![Quadtree Cells](figures/quadtree.png)  | ![Octree Cells](figures/octree.png)  |
```

Comment:
For octree.png, I would try to find a viewpoint that emphasizes that this is a 3D division, at first glance this looked like a quadtree to me.